lib/ui/ClaimableRequest/claimable_request.dart

```diff
@@ -0,0 +1,152 @@
+
+import 'package:blockbill/utils/widgets/custom_button.dart';
+import 'package:flutter/material.dart';
+import 'package:stellar_flutter_sdk/stellar_flutter_sdk.dart' as xlm;
+
+final xlm.StellarSDK sdk = xlm.StellarSDK.TESTNET;
```

Comment:
I have a problem between :3 and :6

If I do not prefix everything with `xlm` the Row class is not detected as a valid widget 😢

Reply:
yes, even I had to use something like that. I couldn't find any better solution.